test/test-apt.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
These tests have a very similar structure. Could we reuse some of the code so there's less duplication?

Reply:
Most of the test code is shared in the `test-r.sh` and `test.R` scripts. These test-apt/yum/zypper scripts are more like bootstrap scripts that differ for the most part (apt vs. yum vs. zypper), and should rarely have to change, so I think it's ok for now.